backend/models/schemas.py

```
@@ -327,3 +327,13 @@ class TokenData(BaseModel):
 #         json_encoders = { ObjectId: str }
 #         arbitrary_types_allowed = True # Allow PyObjectId
 #         populate_by_name = True # To allow using '_id' as 'id'
+
+class ZoneStatus(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)